CppCoreGuidelines.md

```diff
@@ -10314,7 +10314,7 @@ or:
 
 ##### Example
 
-    int a = 7, b = 9, c, d = 10, e = 3;
+    int a = 10, b = 11, c = 12, d, e = 14, f = 15;
```

Comment:
@shaneasd wrote:
> If you were creating constants for hexadecimal translation wouldn't you say `int a = 0xa`? At which point wouldn't you conclude there's no need for a custom constant at all?

You might create local constants for the purpose of array alignment; I've done that several times in my career. (But usually with just the single constant `_` defined to `-1` or `std::nullopt` or something. And usually with macros.)

    static constexpr int a = 10, b = 11, c = 12, d = 13, e = 14, f = 15;
    static const char unpacked_hexdata[] = {
        6, e, 6, f, 7, 7, 2, 0, 6, 9, 7, 3, 2, 0, 7, 4, 6, 8, 6, 5, 2, 0, 7, 4, 6, 9, 6, d, 6, 5, 2, 0,
        6, 6, 6, f, 7, 2, 2, 0, 6, 1, 6, c, 6, c, 2, 0, 6, 7, 6, f, 6, f, 6, 4, 2, 0, 6, d, 6, 5, 6, e,
        2, 0, 7, 4, 6, f, 2, 0, 6, 3, 6, f, 6, d, 6, 5, 2, 0, 7, 4, 6, f, 2, 0, 7, 4, 6, 8, 6, 5, 2, 0,
        6, 1, 6, 9, 6, 4, 2, 0, 6, f, 6, 6, 2, 0, 7, 4, 6, 8, 6, 5, 6, 9, 7, 2, 2, 0, 6, 3, 6, f, 7, 5,
        6, e, 7, 4, 7, 2, 7, 9,
    };

Reply:
fair enough. I guess your new values cover both possibilities. I'm not convinced of the value of the change personally but I see nothing wrong with it.